extra-requirements.txt

```diff
@@ -0,0 +1 @@
+pandas
```

Comment:
I think, you should define version range: pandas>=n.n.n<n+1

Reply:
done